core/src/main/scala/com/criteo/cuttle/Executor.scala

```diff
@@ -150,10 +150,17 @@ case class Execution[S <: Scheduling](
   private[cuttle] var startTime: Option[Instant] = None
   private val cancelListeners = TSet.empty[CancellationListener]
   private val cancelled = Ref(false)
+  /**
+    * An execution with forcedSuccess set to true will have its side effect return a successful Future instance even if the
+    * user code raised an exception or returned a failed Future instance.
+    */
+  private val forcedSuccess = Ref(false)
 
   /** Returns `true` if this [[Execution]] has been cancelled. */
   def isCancelled = cancelled.single()
 
+  def shouldForceSuccess = forcedSuccess.single()
```

Comment:
Is it useful ?

Reply:
It is used at line 664 and is used to separate the atomic boolean `forcedSuccess` from the actual "view" of that object, in the same way `isCancelled` was used.

Reply:
I suggest that we delete it and use `forcedSuccess.single()` at line 664